### ISSUE TITLE
Unable to create map.

### ISSUE THREAD

**Issue by stundzig** [Polymap4/polymap4-p4#50]
After the import of my *locations* data, i navigate into the catalog, find in the *Projects Database* my *locations* dataset and click on it.

I got the attached error message ![bildschirmfoto 2015-11-20 um 11 47 06](https://cloud.githubusercontent.com/assets/1140546/11298064/dd308e5c-8f7c-11e5-9acd-4decdd497ce5.png) 
and the log contains:
```
[INFO]  MdListViewer           : selection: org.eclipse.jface.viewers.SelectionChangedEvent[source=org.polymap.rhei.batik.toolkit.md.MdListViewer@4bf1ceb5]
[INFO]  DefaultAppManager      : CREATE panel: PanelPath[/start/catalog/resource]
[INFO]  UIUtils                : Control: 2030639991, previous variant: batik-panel, new: batik-panel-section-client
[INFO]  UIUtils                : Control: 741056806, previous variant: batik-panel, new: batik-panel-section-client
[INFO]  UIUtils                : Control: 2040190026, previous variant: batik-panel, new: batik-panel-section-client
[INFO]  SvgImageRegistryHelper : SVG -> /Users/stundzig/servers/workspace-arena/.metadata/.plugins/org.polymap.rhei.batik/svg-icons/255-255-255/P24/check.png (24ms)
[INFO]  UIUtils                : Control: 521316054, previous variant: batik-panel, new: batik-panel-fab
[ERROR] StatusDispatcher       : Unable to create map.
java.lang.RuntimeException: Unhandled service type: org.polymap.core.data.rs.RDataStore@43f9d3c7
	at org.polymap.p4.catalog.ResourceInfoPanel$MapDashlet.createMap(ResourceInfoPanel.java:266)
	at org.polymap.p4.catalog.ResourceInfoPanel$MapDashlet$1.lambda$0(ResourceInfoPanel.java:157)
	at org.polymap.p4.catalog.ResourceInfoPanel$MapDashlet$1$$Lambda$174/1457605613.call(Unknown Source)
	at org.polymap.core.runtime.UIThreadExecutor.run(UIThreadExecutor.java:314)
	at org.eclipse.swt.widgets.RunnableLock.run(RunnableLock.java:38)
	at org.eclipse.swt.widgets.Synchronizer.runAsyncMessages(Synchronizer.java:180)
	at org.eclipse.swt.widgets.Display.runPendingMessages(Display.java:1196)
	at org.eclipse.swt.widgets.Display.safeReadAndDispatch(Display.java:1176)
	at org.eclipse.swt.widgets.Display.readAndDispatch(Display.java:1168)
	at org.polymap.rhei.batik.BatikApplication.createUI(BatikApplication.java:122)
	at org.eclipse.rap.rwt.internal.lifecycle.RWTLifeCycle.createUI(RWTLifeCycle.java:172)
	at org.eclipse.rap.rwt.internal.lifecycle.RWTLifeCycle$UIThreadController.run(RWTLifeCycle.java:284)
	at java.lang.Thread.run(Thread.java:745)
	at org.eclipse.rap.rwt.internal.lifecycle.UIThread.run(UIThread.java:105)
[WARN]  StatusDispatcher       : Unable to create map.
java.lang.RuntimeException: Unhandled service type: org.polymap.core.data.rs.RDataStore@43f9d3c7
	at org.polymap.p4.catalog.ResourceInfoPanel$MapDashlet.createMap(ResourceInfoPanel.java:266)
	at org.polymap.p4.catalog.ResourceInfoPanel$MapDashlet$1.lambda$0(ResourceInfoPanel.java:157)
	at org.polymap.p4.catalog.ResourceInfoPanel$MapDashlet$1$$Lambda$174/1457605613.call(Unknown Source)
	at org.polymap.core.runtime.UIThreadExecutor.run(UIThreadExecutor.java:314)
	at org.eclipse.swt.widgets.RunnableLock.run(RunnableLock.java:38)
	at org.eclipse.swt.widgets.Synchronizer.runAsyncMessages(Synchronizer.java:180)
	at org.eclipse.swt.widgets.Display.runPendingMessages(Display.java:1196)
	at org.eclipse.swt.widgets.Display.safeReadAndDispatch(Display.java:1176)
	at org.eclipse.swt.widgets.Display.readAndDispatch(Display.java:1168)
	at org.polymap.rhei.batik.BatikApplication.createUI(BatikApplication.java:122)
	at org.eclipse.rap.rwt.internal.lifecycle.RWTLifeCycle.createUI(RWTLifeCycle.java:172)
	at org.eclipse.rap.rwt.internal.lifecycle.RWTLifeCycle$UIThreadController.run(RWTLifeCycle.java:284)
	at java.lang.Thread.run(Thread.java:745)
	at org.eclipse.rap.rwt.internal.lifecycle.UIThread.run(UIThread.java:105)
```

**Comment by stundzig** [comment]
After a restart of the P4 application, i can see the data.

**Comment by fb71** [comment]
Fixed by #49. Please cross check :D
